internal/ospath/ospath_test.go

```diff
@@ -1,13 +1,20 @@
 package ospath
 
 import (
+	"fmt"
 	"os"
 	"path"
 	"testing"
 
 	"github.com/windmilleng/tilt/internal/testutils/tempdir"
 )
 
+func TestBullshit(t *testing.T) {
```

Comment:
?

Reply:
this either -- was testing weird ospath.Child hanging